### PR TITLE
LIWO-554: north-indicator & scale controls

### PR DESCRIPTION
**Pull request by sjoerdbeentjes**
- Only implements controls, the functionality to toggle them on & off will come in LIWO-496

### CODE REVIEW

File: src/lib/leaflet-utils/map-factory.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import '@/lib/leaflet-hack'
 import mapConfig from '@/map.config'
 import { EPSG_3857 } from '../../lib/leaflet-utils/projections'
 import createCrs from '../../lib/leaflet-utils/create-crs'
+import northIcon from '../../img/north-arrow-2.svg'
```
Comment (SiggyF) [inline src/lib/leaflet-utils/map-factory.js:7]: Is north-arrow  also used?

File: src/lib/leaflet-utils/map-factory.js [hunk 2]
```diff
@@ -154,3 +157,21 @@ function layerControl (layers) {
...
+
+    div.innerHTML = `<img width="34" style="padding:4px" src="${northIcon}" alt="">`
+
+    return div
+  }
+
+  Control.addTo(map)
+
+  return whenReady(Control, (el) => {
+    console.log(el)
```
Comment (SiggyF) [inline src/lib/leaflet-utils/map-factory.js:175]: Remove?

File: src/lib/leaflet-utils/map-factory.js (hunk 1 above)
Comment (SiggyF) [reply src/lib/leaflet-utils/map-factory.js:7]: Could you add this to the img src directly? Why do you import it in js?

File: src/lib/leaflet-utils/map-factory.js [hunk 3]
```diff
@@ -21,8 +22,10 @@ export default function (el, vnode, config) {
   // map.addLayer(baseLayers[INITIAL_BASELAYER])
   map.setZoom(config.zoom || mapConfig.zoom)
 
+  map.addControl(roseControl(map))
```
Comment (SiggyF) [inline src/lib/leaflet-utils/map-factory.js:25]: Was L.rose.control not working properly?

File: src/lib/leaflet-utils/map-factory.js [hunk 4]
```diff
@@ -154,3 +157,21 @@ function layerControl (layers) {
     addListener(el)
   })
 }
+
+function roseControl (map) {
+  var Control = L.control({position: 'topright'})
```
Comment (SiggyF) [inline src/lib/leaflet-utils/map-factory.js:162]: `Control` -> `control`?

**Review by SiggyF — APPROVED** [review]
Some questions.

File: src/lib/leaflet-utils/map-factory.js (hunk 3 above)
Comment (sjoerdbeentjes) [reply src/lib/leaflet-utils/map-factory.js:25]: No it didn't work when imported, and also the provided images were nog satisfactory

File: src/lib/leaflet-utils/map-factory.js (hunk 1 above)
Comment (sjoerdbeentjes) [reply src/lib/leaflet-utils/map-factory.js]: -  changed the image name
- This way we keep things consistent (we import svg's in other files) and leave it up to webpack how to handle the import. This way the SVG can also be optimized and/or be presented as a base64 url

**Pull request merged**